contracts/tg4-engagement/src/contract.rs

```diff
@@ -913,8 +913,26 @@ fn list_members_by_points<Q: CustomQuery>(
 }
 
 #[cfg_attr(not(feature = "library"), entry_point)]
-pub fn migrate(deps: DepsMut, _env: Env, _msg: Empty) -> Result<Response, ContractError> {
+pub fn migrate(
+    deps: DepsMut<TgradeQuery>,
+    _env: Env,
+    msg: MigrateMsg,
+) -> Result<Response, ContractError> {
     ensure_from_older_version(deps.storage, CONTRACT_NAME, CONTRACT_VERSION)?;
```

Comment:
Is this ensure check required? It forces user to update version only to change that parameter.

Reply:
It supports version equality nicely.